apps/designer/app/designer/features/style-panel/shared/property-name.tsx

```diff
@@ -1,39 +1,89 @@
-import { Flex, Label, Tooltip } from "@webstudio-is/design-system";
+import { useState } from "react";
+import type { StyleProperty } from "@webstudio-is/css-data";
+import {
+  Button,
+  Flex,
+  Box,
+  Text,
+  Label,
+  Tooltip,
+  Popover,
+  PopoverContent,
+  PopoverPortal,
+  PopoverTrigger,
+  Separator,
+} from "@webstudio-is/design-system";
+import { UndoIcon } from "@webstudio-is/icons";
 import { useIsFromCurrentBreakpoint } from "./use-is-from-current-breakpoint";
-import { propertyNameColorForSelectedBreakpoint } from "./constants";
-import type { PropertyProps } from "../style-sections";
 
-export const PropertyName = ({ property, label, css }: PropertyProps) => {
+type PropertyProps = {
+  property: StyleProperty | StyleProperty[];
+  label: string;
+  onReset: () => void;
+};
+
+export const PropertyName = ({ property, label, onReset }: PropertyProps) => {
   const isCurrentBreakpoint = useIsFromCurrentBreakpoint(property);
+  const [isOpen, setIsOpen] = useState(false);
+  const isPopoverEnabled = isCurrentBreakpoint;
+
+  const labelElement = (
+    <Label
+      css={{
+        fontWeight: "inherit",
+        padding: "calc($spacing$3 / 2) $spacing$3",
+        ...(isCurrentBreakpoint
+          ? {
+              color: "$blue11",
+              backgroundColor: "$colors$blue4",
+              borderRadius: "$borderRadius$4",
+            }
+          : {
+              color: "$hiContrast",
+            }),
+      }}
+      htmlFor={property.toString()}
+    >
+      {label}
+    </Label>
+  );
+
+  if (isPopoverEnabled) {
+    return (
+      <Flex align="center">
+        <Popover modal open={isOpen} onOpenChange={setIsOpen}>
+          <PopoverTrigger asChild aria-label="Show proprety description">
+            {labelElement}
+          </PopoverTrigger>
+          <PopoverPortal>
+            <PopoverContent align="start" onClick={() => setIsOpen(false)}>
+              <Flex align="start" css={{ padding: "$spacing$3 $spacing$4" }}>
+                <Button onClick={onReset}>
+                  <UndoIcon /> &nbsp; Reset
+                </Button>
+              </Flex>
+              <Separator />
+              <Box css={{ padding: "$spacing$3 $spacing$4" }}>
+                <Text color="hint">
+                  Resetting will revert to initial or inherited value
```

Comment:
Since we are unsetting the property, we are essentially going back to:
- inherited value if there is one
- cascaded value if there is one (when it was defined on a different breakpoint that impacts the current breakpoint)
- preset value if there is one
- browser initial value (https://developer.mozilla.org/en-US/docs/Web/CSS/initial_value)  

If we make it entirely correct, we need to actually check what we would be resetting to:

e.g. we could check if there is an inherited value and if there is: "Resetting will revert to inherited value" and do the same with cascaded, preset and browser initial. 

That's what webflow does, except that they still call both cascaded value and inherited value "inherited".

Reply:
Yes, I left this for another PR caz reset logic alone requires a lot of changes

Reply:
<img width="208" alt="Screenshot 2022-12-10 at 10 23 57" src="https://user-images.githubusercontent.com/52824/206850049-97192f47-6d2e-4044-86b3-5bd009fb418a.png">
<img width="203" alt="Screenshot 2022-12-10 at 10 24 07" src="https://user-images.githubusercontent.com/52824/206850051-2ad7eade-50ec-45b9-a574-78f7e66f1628.png">